terralab/utils.py

```diff
@@ -29,10 +23,55 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         except ApiException as e:
             formatted_message = f"API call failed with status code {e.status} ({e.reason}): {json.loads(e.body)['message']}"
-            LOGGER.error(formatted_message)
+            LOGGER.error(add_blankline_after(formatted_message))
```

Comment:
yea we should make a ticket to add newlines to the end of every `error` log

Reply:
yeah that would be nice. will do

Reply:
https://broadworkbench.atlassian.net/browse/TSPS-374